src/app/tests/suites/certification/Test_TC_GRPKEY_2_1.yaml

```
@@ -85,25 +85,8 @@ tests:
                         EpochStartTime2: "18446744073709551614",
                     }
 
-    - label:
-          "Step 5: TH sends AddGroup Command to DUT on PIXIT.G.ENDPOINT with the
-          the following settings GroupID: 0x0103 GroupName: 'Test Group'"
-      PICS: G.S.C00.Rsp
-      endpoint: Groups.Endpoint
-      cluster: "Groups"
-      command: "AddGroup"
-      arguments:
-          values:
-              - name: GroupID
-                value: 0x0103
-              - name: GroupName
-                value: "Test Group"
-      response:
-          values:
-              - name: "Status"
-                value: 0
-              - name: "GroupID"
-                value: 0x0103
+    #Step 5 Removed in test-plan PR: https://github.com/CHIP-Specifications/chip-test-plans/pull/3425
+    #TH sends AddGroup Command to DUT on PIXIT.G.ENDPOINT with the the following settings GroupID: 0x0103 GroupName: "Test Group"
```

```suggestion
    # Step 5 does not exist in the test plan.
```
That's all we should say here.  People who want to know what the story is with the test plan can then look there.